code/modules/mob/language/outsider.dm

```diff
@@ -115,3 +115,17 @@
 		return 1
 
 	return 0
+
+//for your antag purposes. machine-translatable, if you don't want that use Cult or one of the animal langs
+/datum/language/minbus
+	name = LANGUAGE_MINBUS
+	desc = "The Powers That Be have seen it fit to grace you with a special language that sounds like Russian for some reason."
+	speech_verb = "says"
+	ask_verb = "asks"
+	exclaim_verb = "shouts"
+	colour = "deadsay"
+	key = "r"
+	flags = RESTRICTED | WHITELISTED
```

Comment:
Not 100% sure what the WHITELISTED flag does, but I'm not sure it should be here.

Reply:
yeah WHITELISTED is probably unnecessary but it also doesn't hurt anyone